save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/UserSettingsView.kt

```diff
@@ -120,7 +133,9 @@ abstract class UserSettingsView : AbstractView<UserSettingsProps, UserSettingsVi
                 userInfo = user
                 image = ImageInfo(user?.avatar)
                 userInfo?.let { updateFieldsMap(it) }
-                selfOrganizationWithUserList = organizationDtos
+                selfOrganizationWithUserList = organizationDtos.filter { it.organization.status == OrganizationStatus.CREATED }
```

Comment:
We have 3 identical blocks of code here.

Is it possible to introduce a single `xyz.filterHasStatus(OrganizationStatus)` extension, so that the resulting call site looks like this?

```suggestion
                selfOrganizationWithUserList = organizationDtos.filterHasStatus(CREATED)
```